app/javascript/packs/application_css.scss

```diff
@@ -10,13 +10,25 @@ footer.primary-footer {
 .admin-navbar {
   .navbar-brand img {
     height: 35px;
-    width: 35px;
   }
   .dropdown-toggle {
     cursor: pointer;
   }
+  // I added this because several elements actually would show up above the navbar
+  z-index: 999;
 }
 
 .full-screen-table-wrapper {
   overflow-x: scroll;
 }
+
+@media screen and (min-width: 600px) {
```

Comment:
Rather than manually writing @media queries, I'd recommend using bootstraps [media query mixins](https://getbootstrap.com/docs/4.3/layout/overview/#responsive-breakpoints)